packages/ui/src/lib/Provider/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { NativeBaseProvider, extendTheme } from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+
+const theme = extendTheme({
+  colors: {
+    white: "#FFFFFF",
+    black: "#000000",
+    support: "#4A4C5B",
+    background: "#F3F4F5",
+    primary: {
+      50: "#CAE8FA",
+      100: "#AFDDF7",
+      200: "#95D1F5",
+      300: "#7AC6F2",
+      400: "#60BBEF",
+      500: "#44AEED",
+      600: "#1E9EE9",
+      700: "#1485C7",
+      800: "#106B9F",
+      900: "#0C5077",
+    },
+  },
+  sizes: {
+    // The following sizes are used but not defined in default theme
+    // https://docs.nativebase.io/default-theme#h2-size=
+    13: 52,
+    14: 56,
+    15: 60,
+    17: 68,
+    18: 72,
+    19: 76,
+  },
```

Comment:
🚨 I'm hesitate to add more sizes without a designer.

Reply:
At the moment, we need these sizes for the existing mockup. Maybe we can remove these after the designer redesign the component library?

Reply:
OK, let's comment this in the code then?

Reply:
wait, we are now in the dev phase, once it's on prod, we spend 100x the effort to remove it, is it worthy it?

Reply:
Make sense, how about I use the closest size in default theme to implement the current design?

Reply:
Sounds good